Java/Unique Binary Search Tree II.java

```diff
@@ -45,10 +45,10 @@
  */
 public class Solution {
     public List<TreeNode> generateTrees(int n) {
-        List<TreeNode> rst = new ArrayList<>();
-        if (n <= 0) {
-            return rst;
-        }
+     //   List<TreeNode> rst = new ArrayList<>();
+     //   if (n <= 0) {
+     //      return rst;
```

Comment:
depending on test case. My assumption is to return `[]`, where in `generate`, it returns [null]. I think either way works.